collector/lib/ConnTracker.cpp

```diff
@@ -40,6 +40,24 @@ static const NRadixTree private_networks_tree(PrivateNetworks());
 
 }  // namespace
 
+bool AdvertisedEndpointEquality::operator()(const ContainerEndpoint& lhs, const ContainerEndpoint& rhs) const {
+  if (bool(lhs.originator()) != bool(rhs.originator())) {
+    return false;
+  }
+
+  if (lhs.originator()) {
+    auto& lhs_originator = *lhs.originator();
+    auto& rhs_originator = *rhs.originator();
+
+    // Here is the real difference with the comparator in ContainerEndpointMap, which makes a deep compare of originators. We only focus on process-unique-key.
```

Comment:
Process unique key is not defined in collector, so maybe mention where it is defined. However this does present a problem as these comments couple us more strongly to sensor and central, and we do want to be able to run collector as a stand alone and give people the option to have something other than sensor ingest messages from collector.

Reply:
Also maybe explain this at the top of the operator rather than here.

Reply:
This is a good point ! I updated the explanations to refer to storage::NetworkProcessUniqueKey instead of the blurry Sensor concept.